### ISSUE TITLE
You didn’t join when this client was made 

### ISSUE THREAD

**Issue by Skitttyy** [Gopro336/Guinness-Deobf#3]
You didn’t join when this client was made lel

**Comment by urkkiz225** [comment]
larp

**Comment by Gopro336** [comment]
> You didn’t join when this client was made lel

huh?
I've been here since 2019 kek.
This client is late 2020 at absolute earliest

**Comment by master7720** [comment]
lol

**Comment by blu3-65b** [comment]
> > You didn’t join when this client was made lel
> 
> 
> 
> huh?
> 
> I've been here since 2019 xD.
> 
> This client is late 2020 at absolute earliest

Correct. Bon and I began development in November 2020 as a 32k client for ECME. Thankfully, considering that next to no servers have 32ks anymore, the client is now all but obsolete.

**Comment by master7720** [comment]
Skitttyy you were poping to legacy client on cc

**Comment by blu3-65b** [comment]
> Skitttyy you were poping to legacy client on cc
Put simply, if a client can place and break crystals, people will pop to it. This is demonstrated very well by even MY dogshit AutoCrystal (from back when I actually played the game). It won me fights. Then again, I wouldn't expect a 15yo to know how reason works...

**Comment by master7720** [comment]
> You didn’t join when this client was made lel

who even cares lol
